src/state/languages/actions.js

```diff
@@ -40,31 +40,34 @@ export const fetchLanguages = (page = { page: 1, pageSize: 10 }, params = '') =>
   })
 );
 
-const setLanguageActive = (langCode, active) => (dispatch, getState) => (
+const setLanguageActive = (langCode, active, messageId) => (dispatch, getState) => (
   new Promise((resolve) => {
-    const langObject = getLanguagesMap(getState())[langCode];
+    const langObject = Object.assign({}, getLanguagesMap(getState())[langCode]);
```

Comment:
why not use the object spread operator (e.g. `{ ...  getLanguagesMap(getState())[langCode] }`) instead of using `Object.assign`?

Reply:
its same and Assign is more expressive the purpose, but agree, spread operator is nicer. Changed.

Reply:
yup its actually the same. but most people here prefer es6 syntax.